Nodejs/Product/Npm/SPI/NodeModules.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.NodejsTools.Npm.SPI {
     internal class NodeModules : AbstractNodeModules {
         public NodeModules(IRootPackage parent, bool showMissingDevOptionalSubPackages) {
             var modulesBase = Path.Combine(parent.Path, "node_modules");
-            if (Directory.Exists(modulesBase)) {
+            if (modulesBase.Length <= 224 && Directory.Exists(modulesBase)) {

```

Comment:
224 because it is 248 - length of "node_modules\.bin" (?) and some room for the nested dependency name. If I would more precise, I have to refactor much more and I under assumption that if you on Windows and hit MAX_PATH limit you are already in world of pain, and if I miss some edge package, nothing wrong happens. I believe that easier to make this work 90% of time, and then workout edge cases for others, since this is really helpful during debugging.